bigchaindb/common/schema/transaction_validator_election_v2.0.yaml

```diff
@@ -43,6 +45,7 @@ properties:
           - node_id
           - public_key
           - power
+          - seed
```

Comment:
We don't have to make it required.

Reply:
Should it be required for the validator election?

Reply:
We do not have to require it there either.

Reply:
It's generated automatically on creation of the election, so it will always be present, regardless. But I can remove the required tag, anyway.